src/util/misc.cc

```diff
@@ -245,7 +245,7 @@ std::vector<int> CSVToVector(const std::string& csv) {
     }
     try {
       values.push_back(std::stoi(elem));
-    } catch (std::exception) {
+    } catch (std::invalid_argument const&) {
       return std::vector<int>(0);
```

Comment:
Can you move the const before the type? Or just apply clang-format and it will do it for you.